src/loaders/boardLoader.ts

```diff
@@ -0,0 +1,24 @@
+import { cardData } from "../gameFunctionality/gameInitialization"
+import rootStore from "../rootStore"
+import { clueObj, team } from "../routes/BoardGame"
+import { gamePropertiesObj } from "../types/gamePropertiesObj"
+const { gamePropertiesStore } = rootStore
+export const boardLoader = (data: gamePropertiesObj) => {
```

Comment:
we can't set all the properties all the time' we need to import the "set" function from the server